test_istgt.sh

```diff
@@ -454,15 +454,17 @@ run_read_consistency_test ()
 		return
 	fi
 
-	write_data 0 41943040 512 "/dev/$device_name" $file_name
+	write_data 0 41943040 4096 "/dev/$device_name" $file_name
```

Comment:
Any reason why the block size was changed?

Reply:
The consistency test was failing as the block size was different in two commands. cc @mynktl

Reply:
@mynktl  Can you explain on these. How did the tests passed earlier?

Reply:
In this particular test, we are duplicating stream/writes to the iscsi volume and local file, and checking if both are having the same data or not.

Initially, we are writing ~40Mb of data to device and local files using 512 blocksize. and in the second stage, we are writing ~10Mb using 4096 blocksize. In both stages, we are writing from offset 0.

Writing 40Mb of data will result in an error due to block size mismatch. Although some initial writes may have succeeded for a local file in the first stage, the second write of 10Mb will make sure that both have the same data.